ignite/distributed/comp_models/base.py

```diff
@@ -143,9 +143,6 @@ def get_num_nodes(self) -> int:
     def get_node_rank(self) -> int:
         return 0
 
-    def is_distributed(self) -> bool:
```

Comment:
Question: why was this removed? There may be a good reason, but I think now the contract enforced by the interface is more implicit.

Reply:
@erip it was discussed in a gdoc about idist API and we thought that it is sort of redundant and what to call `distributed`... So, now we check `get_world_size() > 1`.